src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -279,8 +279,8 @@ internal string CasingText
 		{
 			get => CharacterCasing switch
 			{
-				CharacterCasing.Lower => Text.ToLower(),
-				CharacterCasing.Upper => Text.ToUpper(),
+				CharacterCasing.Lower => Text.ToLowerInvariant(),
+				CharacterCasing.Upper => Text.ToUpperInvariant(),
```

Comment:
Ok. Undone.

Reply:
Why did you add this change originally? Which issue did you find?

Reply:
When i was researching why the tests are failing, i found something related to that. But cant remember if it was a comment inside the uno code itself or in the discussion on another thread. 
As the test was using Invariant, i thought it could be better to use that.

And of course, i will squash the many commits when the test pass.